### DISCUSSION TITLE
trying here isms again

### DISCUSSION THREAD

**Discussion by igormpc** [tehn/isms#6] [General]
hi brian
trying isms again here:

i had to add to Makefile line 14
-I/usr/include/libevdev-1.0
to find libevdev here on archlinux

working ok no errors with 40h :)
sdl ok, kb sending msg to sc, drawing lines with random pos
sc ok, receiving msg from sdl/kb + 40h, sending osc msg to repl /hello

some questions:
should leds on 40h toggle on and off? after one led gets on it didnt get off playing it again...

how can i use remote lua input via UDP? 
on "set up your editor to send to this port" --
can i use neovim for it?
can you detail how can i config it for it? 
(i suppose i can use for vimrc keybinding after...)

thx in advance! :)

**Comment by tehn** [comment]
neovim should work with that same line i specify in the readme, ie try

```
map <C-a> :silent .w !xargs -0 echo -n > /dev/udp/localhost/11001
```

**Reply by tehn** [reply]
also heads up that i'm really changing a lot in this codebase. like i said before, this is not really ready for public use, unless you're willing to really get into reading the code.
